build/images/kubevirt-cloud-controller-manager/Dockerfile

```diff
@@ -1,6 +1,10 @@
-FROM alpine:3.11.2
+FROM --platform=linux/amd64 golang:1.18.3-buster AS builder
```

Comment:
We should use golang 1.17 image since we have this version in go.mod.

However, I would go with go-runner image that wraps stderr and stdout. We are using klog in this project and there is a KEP to remove flags like `--log-file` [link](https://github.com/kubernetes/enhancements/blob/master/keps/sig-instrumentation/2845-deprecate-klog-specific-flags-in-k8s-components/README.md). Additionaly the whole k8s ecosystem is moving towards that solution.

So maybe it's better to use image k8s.gcr.io/build-image/go-runner:v2.3.1-go1.17.11-bullseye.0

Reply:
ah yes, you're right I overlooked. But for the container itself maybe we could use the go-runner anyway?

Reply:
Additionaly, is there any reason why you used `buster` for the build stage?

Reply:
> But for the container itself maybe we could use the k8s.gcr.io/build-image/go-runner:v2.3.1-go1.17.11-bullseye.0 anyway?

I will change the container image and will update

> Additionaly, is there any reason why you used buster for the build stage?

I saw some examples in other repo which use this for the build container, no specific reason

Reply:
@mfranczy as we discussed, changed the base images:
container image to `registry.redhat.io/ubi9/ubi-micro`
build image to `golang:1.17.11`
Please take a look